e2etest/zt_preserve_access_tier_test.go

```diff
@@ -26,6 +26,19 @@ import (
 	"testing"
 )
 
+//func TestTier_UploadCold(t *testing.T) {
+//	RunScenarios(t, eOperation.Copy(), eTestFromTo.Other(common.EFromTo.LocalBlob()), eValidate.Auto(), anonymousAuthOnly, anonymousAuthOnly, params{
+//		recursive:             true,
```

Comment:
Can we add an item in our backlog to remind us to bring this test back once the service is live?

Reply:
Backlog items tend to get lost in the weeds, but yes, we should do this. I think we should keep it in the current release and just push it back another step each time while we wait so it gets brought up in the sync-up, rather than stick it in "el backlog del backlog backlog" for it to get lost to time.